ada_2_fhir-r4/zibs2024/ada_instance/ADA_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+# ADA guidelines
+
+In this folder instances of the zib scenarios are added in ADA format, which are used to test the conversion from zibs to FHIR resources.
+
+## ADA environment
+All ADA instances are created via the ADA environment accessible at `https://decor.nictiz.nl/art-decor/ada-data/projects/zib2024-prerelease/views/zorginformatiebouwstenen_zib_2024_index.xhtml`. Make sure to add these while being logged in with the _zibs2024ada_ account. This ensures that instances that have been created in the context of this project can be easily distinguished from other instances, while also making it easy for anyone in the zibs2024 project to update existing instances (without copying existing ones made by other accounts).
+
+## ADA instance conventions
+* The example instances available on the respective zib's page are manually recreated as-is in an ADA instance. Both `title` and `id` of these ADA instances SHOULD have the form `zib-[English zib name]-0x`, where `x` denotes the 'sequence number' of the zib example (note that in most cases there is only a single example instance for each zib).
```

Comment:
The ADA instances in the zibs2020 project currently have names of the form _nl-core-[English zib name]-0x_, but I think prefixing with _zib_ is better in order to make clear these are zib example instances (and thus do not contain additional data).

For the FHIR instances we might consider to change the prefix to _nl-core_ during the conversion, to indicate these FHIR instances represent instances conforming to the respective nl-core profile.

Reply:
Agree that it would be better, but it would be even better if the ADA instances weren't checked in to the HL7-repo at all (because why would they if ART-DECOR is the actual source). So rather than investing in half an improvement (well, as I see it), it'd suggest to just keep the nl-core-prefix until we have something that's actually better.